.circleci/config.yml

```diff
@@ -102,6 +102,7 @@ commands:
             conda activate esmvaltool
             dpkg -l | tee /logs/versions.txt
             conda env export | tee /logs/environment.yml
+            conda list  # print to screen too
```

Comment:
the command above should already print this to stdout

Reply:
no sorry, it does, but the formatting is cluttered, and it's missing the channel info (yes, it ought to be conda-forge only but I like to see everything) :smiley:

Reply:
The formatting looks like what the usual environment.yml file would look like, including the channel (`conda-forge`). You can find an example of the output [here](https://app.circleci.com/pipelines/github/ESMValGroup/ESMValCore/13039/workflows/6ae23443-2a6e-4f2f-be60-ebc2f1b119a5/jobs/54086) under the `Log versions` step.

Reply:
OK nevermind, I was more keen on figuring out clean what's going on with the devel deps test failing, the conda list was more of a side-show :grin: